tools/project_manager/remote_file_manager.php

```diff
@@ -683,11 +726,14 @@ function do_showmove()
     // Remove first element (which is $home_path itself)
     unset($user_subdirs[0]);
     $valid_target_dirs = array_merge($valid_target_dirs, $user_subdirs);
-    natcasesort($valid_target_dirs);
 
-    // Allow users to tranfer files to Commons too
-    if(!in_array($commons_dir, $valid_target_dirs))
-        array_unshift($valid_target_dirs, $commons_dir);
+    // And all subdirectories in the Commons directory
+    $common_subdirs = searchdir($commons_dir, 2, "DIRS");
```

Comment:
What does the "And" in the comment refer to? To me it makes most sense following the "Allow users to transfer files to Commons too", but it looks like that was deleted.